src/types.ts

```diff
@@ -36,11 +36,16 @@ export interface Column<TRow, TSummaryRow = unknown> {
   sortable?: boolean;
   /** Sets the column sort order to be descending instead of ascending the first time the column is sorted */
   sortDescendingFirst?: boolean;
-  // TODO: Should we return an object to make it more powerful?
-  // { isEditable?: boolean, initialInput?: unknown } | undefined
-  unsafe_onCellInput?: (event: React.KeyboardEvent<HTMLDivElement>, row: TRow) => unknown;
   /** Editor to be rendered when cell of column is being edited. If set, then the column is automatically set to be editable */
   editor?: React.ComponentType<EditorProps<TRow[keyof TRow], TRow, TSummaryRow>>;
+  editor2?: React.ComponentType<Editor2Props<TRow, TSummaryRow>>;
+  editor2Options?: {
+    createPortal?: boolean;
+    singleClickEdit?: boolean;
+    commitOnOutsideClick?: boolean;
```

Comment:
Let's remove or comment this out if it won't be implemented in this PR.

Reply:
It was a quick implementation so I added it